front/pages/[user]/a/[sId]/execute/index.jsx

```diff
@@ -94,7 +94,7 @@ function ExecuteOutputLine({
             <ChevronDownIcon className="text-gray-400 h-4 w-4" />
           )}{" "}
           <div className="inline">
-            <span class="rounded-md px-1 py-0.5 bg-gray-200 font-medium text-sm">
+            <span className="rounded-md px-1 py-0.5 bg-gray-200 font-medium text-sm">
```

Comment:
@spolu  unrelated change but this was showing an error in the console so I assumed it was a typo

Reply:
definitely is :+1: